openfisca_us/variables/gov/states/mo/tax/income/credits/mo_property_tax_credit/mo_property_tax_credit_amount.py

```diff
@@ -0,0 +1,38 @@
+from openfisca_us.model_api import *
+
+
+class mo_property_tax_credit_demographic_tests(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MO property tax credit demographic eligiblity test"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://dor.mo.gov/forms/MO-PTS_2021.pdf"
+    defined_for = StateCode.MO
+
+    def formula(tax_unit, period, parameters):
+        #currently not including railroad retirement or veterans benefits. 
+        rent_or_own = tax_unit("mo_property_tax_credit_rent_or_own", period)
+
+        #determine if the tax unit head and spouse co-habitate
+        lives_separately = tax_unit("lives_separately", period)
+        living_arrangement = "SINGLE" if lives_separately > 0 else "JOINT"
+
+        income_threshold = parameters(period).gov.states.mo.tax.credits.mo_property_tax_credit[rent_or_own][living_arrangement]
```

Comment:
Reorganize parameters as following, then reuse `p`:
```suggestion
        p = parameters(period).gov.states.mo.tax.credits.property_tax
        income_limit = p.income_limit[rent_or_own][living_arrangement]
```